team_admin/discord.md

```diff
@@ -25,4 +25,6 @@ Along with the common channels, there are some channels visible only to members
 
 `team-ABC` (where `ABC` is your TLA) is a channel only your team (and blue-shirts) can see. Here you can converse among your team, and get help from blue-shirts without giving away too much of your strategy. These channels should be considered your own, and can be used to discuss anything competition related for your team.
 
+When asking for support, it's best to ask in public rooms unless there's a specific reason not to. This not only allows other teams who may be struggling with the same issue to benefit, but also allows them to contribute to a solution and discuss potential improvements.
+
 If your school / institution has multiple teams (`ABC2` etc), there will be an additional channel for each of those (`teams-ABC2`). Remember to try and keep conversations specific to the team - just because the team comes from the same school / institution, doesn't make them any less your rival.
```

Comment:
If the channels are private and you can only be in one of them, does the second part of this make sense?

Reply:
This wasn't the plan as I interpreted it? I thought we'd have different roles for the 2 teams, but keep one channel.

Maybe we should just drop this line, we might end up swapping them out based on what teams ask for

Reply:
If the channels are private then I think it's important that they're private _per-team_. Otherwise it definitely creates an imbalance between teams from single-team-schools and multi-team-schools. I'm not sure what the benefit of school-level-private channels would be?

Reply:
We definitely want different channels per team (and thus need different roles).

in hindsight I agree we don't need the school-level private channel is for multi-team schools.

Reply:
https://github.com/srobo/docs/pull/150/commits/9142d58583670b7a6987a6ed3c094cccd30bbcf4